styles/stylesheet.css

```diff
@@ -0,0 +1,391 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    display: flex;
+    flex-direction: column;
+}
+
+header {
+    display: flex;
+    flex-direction: column;
+}
+
+h1, h2, label{
+    font-family: 'Neuton', serif;
+    color: #113c60;
+}
+
+p, a{
+    font-family: 'Work+Sans', sans-serif;
+}
+
+.header-logo-checkout-overlay {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    z-index: 2;
+}
+
+.header-logo-wrapper {
+    display: flex;
+    flex-direction: column;
+    margin-top: 3em;
+    margin-bottom: 10em;
+    margin-left: 5em;
+}
+
+.header-logo-heading {
+    color: #86b349;
+    font-family: serif;
+    font-style: italic;
+    font-weight: normal;
+}
+
+.header-checkout-wrapper {
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-template-rows: 2em 3em;
+    margin-right: 5em;
+    margin-top: 3em;
+    margin-bottom: 10em;
+}
+
+.header-checkout-icon {
+    width: 1.5em;
+    grid-column-start: 1;
+    grid-column-end: 2;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    justify-self: end;
+    align-self: end;
+}
+
+.header-checkout-cart {
+    color: #ffffff;
+    grid-column-start: 2;
+    grid-column-end: 3;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    justify-self: end;
+    align-self: baseline;
+}
+
+.header-checkout-link {
+    background-color: #86b349;
+    border-radius: 3px;
+    padding: .5rem 3rem;
+    grid-column-start: 1;
+    grid-column-end: 3;
+    grid-row-start: 2;
+    place-self: center;
+    text-decoration: none;
+    color: #ffffff;
+}
+
+.header-background-image-container {
+    position: absolute;
+    max-width: 100%;
+    z-index: -1;
+    background-color: #000000;
+}
+
+.header-background-image {
+    float: right;
+    max-width: 80%;
+}
+
+.header-nav{
+    background-color: #dad8d2;
+    color: #113c60;
+    font-weight: bold;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    padding: 0px 210px;
```

Comment:
You should no longer be using px for padding, font size, etc. Use em, rem or % instead.